docs/getting-started-scalardl-auditor.md

```diff
@@ -0,0 +1,545 @@
+Please refer to the `Step 1` ~ `Step 7` of [Getting Started with Helm Charts (Scalar DL Ledger)](./getting-started-scalardl-ledger.md) to create a Ledger environment.  
+To enable Auditor configuration in the Scalar DL Ledger, set `true` to `ledger.scalarLedgerConfiguration.ledgerAuditorEnabled` in `Step 7` of the above guide.  
+
+After this step, we will create (add) a Scalar DL Auditor environment into the above Scalar DL Ledger environment.  
+Also, after this step, we call `Scalar DL Ledger` as `Ledger` and `Scalar DL Auditor` as `Auditor`.  
+
+## Step 2. Start Cassandra container for Auditor
+
+In this guide, we use Apache Cassandra as backend storage of the Auditor, it will be started on the same network of Auditor (Pod on minikube) to enable proper communication.
+
+1. Start Cassandra container for Auditor on the Docker Network `minikube`.
+   ```console
+   $ docker run --name cassandra-auditor --network minikube -d cassandra:3.11
+   ```
+   * Note: 
+       * The Docker Network `minikube` was created by the `minikube start --driver=docker` command that we ran in Step 1.
+       * You need to specify the Cassandra version (tag) that the Scalar DB that is used in Scalar DL supports.
+           * You can see the version of Scalar DB that is used in each Scalar DL in the [build.gradle](https://github.com/scalar-labs/scalar/blob/master/build.gradle) file (see the value of `scalarDbVersion`).
+           * Also, you can see the Cassandra version that is supported by Scalar DB in [this document](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md).
+
+1. Check the Cassandra containers are running.
+   ```console
+   $ docker ps -f name=cassandra-auditor
+   CONTAINER ID   IMAGE            COMMAND                  CREATED          STATUS          PORTS                                         NAMES
+   2a1d2770bf72   cassandra:3.11   "docker-entrypoint.s…"   6 seconds ago    Up 4 seconds    7000-7001/tcp, 7199/tcp, 9042/tcp, 9160/tcp   cassandra-auditor
+   ```
+
+1. Check the status of Cassandra.
+   ```console
+   $ docker exec -t cassandra-auditor cqlsh -e "show version"
+   [cqlsh 5.0.1 | Cassandra 3.11.11 | CQL spec 3.4.4 | Native protocol v4]
+   ```
+   It may take some time to start Cassandra in the container. So, if this command return error, wait a moment and then re-run it.
+
+## Step 3. Create key/certificate files for Auditor
+
+In this section, we will create key/certificate files for Auditor.  
+
+* Note: 
+    * In this guide, we will use self-sign certificates for the test. However, it is strongly recommended that these certificates NOT be used in production.
+    * We assume that you already made a working directory and create key/certificate files for Ledger and Client in the [Getting Started with Helm Charts (Scalar DL Ledger)](./getting-started-scalardl-ledger.md).
+
+1. Change working dir to `certs/`.
```

Comment:
Ditto

Reply:
I fixed in 8538ab98e531f0a02391952adbc58601cac0ae0b.